src/Raven.Studio/wwwroot/App/views/database/indexes/deleteIndexesConfirm.html

```diff
@@ -7,11 +7,26 @@
             <h4 class="modal-title" data-bind="text: title"></h4>
         </div>
         <div class="modal-body">
-            <p data-bind="visible: indexNames.length === 1">You're deleting <strong data-bind="text: indexNames[0]"></strong></p>
-            <p data-bind="visible: indexNames.length > 1">You're deleting <strong data-bind="text: indexNames.length"></strong> indexes</p>
-            <ul data-bind="visible: indexNames.length > 1, foreach: indexNames" style="max-height: 200px; overflow-y: auto;">
-                <li data-bind="text: $data"></li>
+            <p data-bind="html: subTitleHtml"></p>
+            <ul data-bind="foreach: indexesInfoForDelete()" style="max-height: 200px; overflow-y: auto;">
+                <li class="padding padding-xs">
+                    <strong data-bind="text: $data.indexName"></strong>
+                    <small data-bind="if: $data.reduceOutputCollection">
+                         <!-- The span code below is formatted like this becuase otherwise we get extra spaces in ui -->
```

Comment:
remove this comment and simply concat '(' with second span text?

Reply:
done